website/src/documentation/programming-guide.md

```diff
@@ -367,8 +367,13 @@ public static void main(String[] args) {
 }
 ```
 ```py
-{% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:model_pcollection
-%}
+with beam.Pipeline(options=PipelineOptions()) as p:
```

Comment:
I believe our docs work by updating the /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py instead of directly updating docs.

That allows us to be able to run on snippets and keep them uptodate.

@davidcavazos could help with updating the snippets and reviewing this PR.